ecr/method.ecr

```diff
@@ -18,9 +18,6 @@ def <%= method_identifier %>(<% render_args_declaration -%>) <%= method_return_t
   <% end -%>
 
   # Return value handling
-  <% if method.constructor? && method.caller_owns.none? -%>
-    LibGObject.g_object_ref_sink(_retval)
-  <% end %>
```

Comment:
generated GObject constructors already take care of sink float references, so this wasn't needed.